app/uk/gov/hmrc/gform/submission/handlebars/HandlebarsTemplateProcessorHelpers.scala

```diff
@@ -421,10 +421,10 @@ class HandlebarsTemplateProcessorHelpers(
   def isNotSuccessCode(code: Integer): CharSequence = log("isNotSuccessCode", code)(condition(!isSuccCode(code)))
   private def isSuccCode(code: Integer): Boolean = code != null && code >= 200 && code <= 299
 
-  def desCurrentDate: CharSequence = condition(DateTimeFormatter.ISO_LOCAL_DATE.format(timeProvider.localDateTime))
-  def currentDate: CharSequence = condition(DateTimeFormatter.BASIC_ISO_DATE.format(timeProvider.localDateTime))
+  def desCurrentDate: CharSequence = condition(DateTimeFormatter.ISO_LOCAL_DATE.format(timeProvider.localDateTime()))
+  def currentDate: CharSequence = condition(DateTimeFormatter.BASIC_ISO_DATE.format(timeProvider.localDateTime()))
```

Comment:
Now the obvious question is why are these stuck in UTC, do we expect that?

Reply:
I thought that by sticking to UTC we make the server's and local environment consistent. So that localDateTime by default provides the local time on the server.
And only in some cases we can use the same function to get local date time in other time zones. We use it only once for `submittedDate`

Reply:
But the question is: Do we want for handlebar templates to stay stuck in UTC or should they produce BST dates?

The fact that implementation is using `timeProvider.localDateTime` I tend to believe that the intention was to use BST and not to stick to UTC.

Reply:
Ok, let's go back to to initial implementation. We should also modify all `LocalDate.now` and `LocalDateTime.now` accordingly to answer your initial question.